main.tf

```diff
@@ -8,8 +8,28 @@ module "default_label" {
   tags       = "${var.tags}"
 }
 
+module "task_role_label" {
+  source     = "git::https://github.com/cloudposse/terraform-terraform-label.git?ref=0.1.3"
+  attributes = ["${compact(concat(var.attributes, list("task", "role")))}"]
```

Comment:
I think we should remove `role` from the name since it's disambiguated by the resource type